### PR TITLE
add sql scripts

### PR DESCRIPTION
**Pull request by totekuh**

### CODE REVIEW

**Comment by mtalimanchuk** [comment]
Let's set the config for an aws rds right away and then I'll merge.

File: app.py [hunk 1]
```diff
@@ -12,23 +12,16 @@
...
+    parser.add_argument('-p', '--port', dest='port', default=5000,
                         help='Optional. Port to listen for the server. Default is 5000/tcp')
-    parser.add_argument('-l', '--logging', dest='log_level',
+    parser.add_argument('-l', '--logging', dest='log_level', default=logging.INFO,
+                        choices=['INFO', 'DEBUG', 'WARN', 'ERROR'],
                         help='Optional. Logging level to use in application. '
                              'Can be one of the following: [INFO, DEBUG, WARN, ERROR]. Default is INFO.')
 
-    options = parser.parse_args()
-    if not options.ip:
```
Comment (totekuh) [inline app.py:16]: why did you remove this?

Comment (totekuh) [reply app.py:16]: nevermind

**Review by totekuh — COMMENTED** [review]
> Let's set the config for an aws rds right away and then I'll merge.

let's implement all dao classes and then merge this pr?

**Comment by mtalimanchuk** [comment]
No, please, let's not make it complicated. We need a wrapper to create a connection and execute some queries with it and basic crud functions
Look at this piece and tell me if we can do it as easily: https://pastebin.com/v2Wd3y1m

**Comment by totekuh** [comment]
base dao are ready. please review them

**Comment by mtalimanchuk** [comment]
Please, remember. We don't need a scalable database functionality. Just a few pre-defined queries with certain parameters
